api-tests/core/content-type-builder/collection-type.test.api.js

```diff
@@ -192,21 +192,11 @@ describe('Content Type Builder - Content types', () => {
                 name: 'ValidationError',
                 path: ['contentType', 'displayName'],
               },
-              {
```

Comment:
Just a note here: these are being removed because they shouldn't have been here to begin with; the only reason "undefined" was listed in the names of content types is because there was a bug.